vendor/blockchain_contracts/src/ethereum/rfc003/ether_htlc.rs

```diff
@@ -0,0 +1,167 @@
+use crate::{
```

Comment:
I did. Please note that I did not touch the contract code in comit_node so you may want to check out the branch and just do a diff between both `ether_htlc` files.

The change I did is that I created `compile_template_to_hex` that is used to calculate the offset and re-used it in `compile_to_hex` to avoid code duplication.

Sorry it's a pain to review, I am open to suggestions...